src/utils/api.js

```diff
@@ -35,3 +35,7 @@ export const postNewComment = (article_id, newComment) => {
     .post(`/articles/${article_id}/comments`, newComment)
     .then((res) => res.data);
 };
+
+export const deleteCommentById = (comment_id) => {
+  return api.delete(`/comments/${comment_id}`).then((res) => res.data);
+};
```

Comment:
There's a console log somewhere though not in this PR 🕵️ if you could please get rid of that
(think it's in the NewCommentForm component)